src/init/starship.bash

```diff
@@ -12,6 +12,8 @@
 # drawn, and only start the timer if this flag is present. That way, timing is
 # for the entire command, and not just a portion of it.
 
+STARSHIP_EXE=::STARSHIP::
```

Comment:
Please remove this change.

Reply:
Sorry, but is there a particular reason why we need 5 different instances of the binary's full path (which replaces the `::STARSHIP::` here)?

Reply:
I don't think a new `STARSHIP_EXE` env var is a big improvement over that, and I'd prefer not to pull unrelated changes into this PR.